PowerUp/app/src/main/java/powerup/systers/com/MapActivity.java

```diff
@@ -53,6 +53,7 @@ public void onClick(View v) {
 			} else {
 				Intent myIntent = new Intent(MapActivity.this, CompletedScene.class);
 				startActivityForResult(myIntent, 0);
+				finish();
```

Comment:
If it is not called, this map activity remains active which causes two map activities to run in the background when we try to come back from completed scene, I tried it and it was leading to reappearance of map activity